app-codepipeline/vars.tf

```diff
@@ -18,6 +18,17 @@ variable "aws_region" {
   }
 }
 
+variable "repo_connection_arn" {
+  type        = string
+  description = "ARN of the AWS codestar connection to use to access the repo."
+  validation {
+    condition     = can(regex("^arn:aws:codestar-connections:\\w+(?:-\\w+)+:\\d{12}:connection\\/[A-Za-z0-9]+(?:-[A-Za-z0-9]+)+$", var.repo_connection_arn))
+    error_message = "Invalid AWS codestar connection ARN"
+  }
+  # "doorway-github-connection" which has been auhtorized for metrotranscom/doorway and metrotranscom/doorway-infra
```

Comment:
Let's leave the default out.  This is a highly specific value that should always be explicitly set

Reply:
Done.